modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -201,18 +201,23 @@ def build_contentions(data)
           end
         end
 
-        def get_phase_type_indicator_array(data)
-          return if data[:benefit_claim_details_dto][:phase_type_change_ind].nil?
+        def current_phase_back(data)
+          if data&.dig(:benefit_claim_details_dto, :bnft_claim_lc_status)&.[](0)&.dig(:phase_type_change_ind).nil?
```

Comment:
Not sure if this is the best way to test for the presence of the phase_type_change_ind here?

Reply:
hmmm... I want to make this more railsy but I don't have thoughts currently. it would have to be broken up in to multiple lines / checks and I don't know if I like that any better. 

Probably would make it easier to read and maintain though 🤔 ....